config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  root 'users#index'
   # Defines the root path route ("/")
   # root "articles#index"
+  
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :create]
+  end
```

Comment:
- [ ] Please make sure that your code only handles the 4 required URLs as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Make%20sure%20that%20you%20added%20code%20to%20handle%20only%20the%204%20required%20URLs.). You added an extra route for post creation which does not exist in the requirement of that project. Kindly remove it.

Reply:
Fixed.